<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="locBox">
       <property name="title">
        <string>Lokomotiven</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="locImage">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="locList"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="bbtBox">
       <property name="title">
        <string>BBT</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="bbtTable"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="calculateBBT">
          <property name="text">
           <string>Berechnen</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>295</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionOpenWorkspace"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="toolTip">
    <string>Beenden</string>
   </property>
   <property name="statusTip">
    <string>Beendet die Anwendung</string>
   </property>
  </action>
  <action name="actionOpenWorkspace">
   <property name="text">
    <string>&amp;Arbeitsbereich öffnen</string>
   </property>
   <property name="iconText">
    <string>Arbeitsbereich öffnen</string>
   </property>
   <property name="statusTip">
    <string>Öffnet einen existierenden Rocrail Arbeitsbereich</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
